packages/core/src/modules/oob/OutOfBandModule.ts

```diff
@@ -67,19 +67,28 @@ export class OutOfBandModule {
   /**
    * Creates an out-of-band message and adds given agent messages to `requests~attach` attribute.
    *
-   * If `handshake` is true, creates new connection record and use its keys for out-of-band message that works as a connection invitation.
-   * It uses discover features to find out what handshake protocols the agent supports.
+   * If you want to create a new connection you need to set `handshakeProtocol` to one of the supported
+   * protocols from `HandhsakeProtocol`, for example:
    *
-   * @param config Optinal attributes contained in out-of-band message
+   * ```ts
+   *  const config = {
+   *    handshakeProtocol: HandshakeProtocol.DidExchange
+   *  }
+   *  const message = outOfBandModule.createMessage(config)
+   * ```
+   *
+   * Then, the out-of-band will use its keys and will work as a connection invitation.
+   *
+   * @param config Configuration and other attributes of out-of-band message
    * @param messages Messages that will be sent inside out-of-band message
-   * @returns Out-of-band message and connection record if it has been created.
+   * @returns Out-of-band message and optionally connection record created based on `handshakeProtocol`
```

Comment:
Updated. Now, if `handshake` is set it assigns handhsake protocols by default and it's possible to override it with `handshakeProtocols` array.

I still need to figure out how to avoid creating the connection record with a particular protocol. But that's a different topic being discussed elsewhere.